Assets/ConnectApp/Redux/Reducers/Reducer.cs

```diff
@@ -863,6 +863,24 @@ public static class AppReducer {
                     break;
                 }
 
+                case UserLicenseMapAction action: {
```

Comment:
here we do only add/modify operations and won't remove the entry which is in the old userLiscenseMap but not in the new map. is it ok?